### ISSUE TITLE
Info list particle could use pagination Feature

### ISSUE THREAD

**Issue by tlrcollette** [gantry/gantry5#1139]
I love this info list in gantry 5. I have an already lengthy list that really could use pagination. The list has the potential to be over a hundred entries. I'll have to abandon the infolist in favor of other technology if this feature is not included in a future release.

ref: http://americankillers.org/cms/index.php/features/murder-mansions-and-murder-houses

Any help would be appreciated. Thanks. :-)

![rt-question1](https://cloud.githubusercontent.com/assets/17143621/12919257/4c243736-cf11-11e5-8955-0110bd853c1c.jpg)

**Comment by mahagr** [comment]
I kind of disagree here as particles aren't really meant to replace content which is coming from CMS as there is really no data storage layer in theme itself -- even with pagination all data would be fetched.

There are some plans to allow particles to fetch data from platform (Joomla articles etc), but that said I kind of like more if the data was editable easier than modifying every entry as separate article.

**Comment by w00fz** [comment]
I agree with Matias here. What you are doing there is borderline between Theme Framework and CMS job. There are also other implications involved from a UI/UX aspect.

1. With pagination you won't be able to easily sort items between pages
2. With the liberty of adding a huge amount of items, finding, editing and managing all of them will become very hard, defeating the purpose of simplicity and, as said earlier, leaving the boundaries of what a Theme Framework job should be.
3. The moment we add pagination, we would basically encourage this use of collections which we really don't want to do. Collections are very powerful but should be used in a limited and controlled way.

Don't get me wrong, I think it's pretty cool that you found your sweet spot for adding content by using  Collections, I just think pagination is not something we want to add as it goes against our idea of collections.

That said, I think you shouldn't be penalized for using collections this way, if it suits you, for this reason I'm going to add at least a maximum height, to make the collection scrollable, without taking the whole height of the window (or more!).

![collections-overflow](https://cloud.githubusercontent.com/assets/11734/13192354/d27b5f9c-d71f-11e5-93f5-314c1a4f6126.png)
